src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -51,22 +52,10 @@ class Riddle {
 		return this.casts.filter(event => event.action === this.data.actions.THE_FORBIDDEN_CHAKRA.id).length
```

Comment:
I know you mentioned RoF needs a rewrite, but just wanted to point out that Enlightenment should be included here.